test/helpers.ts

```diff
@@ -11,7 +11,9 @@ import {
 	getScopedSelector,
 	getLatestVersionTag,
 	preventPrCommitLinkLoss,
-	prCommitUrlRegex
+	prCommitUrlRegex,
+	prCompareUrlRegex,
+	preventPrCompareLinkLoss
```

Comment:
Let's move them together so nobody will confuse them

```suggestion
	preventPrCompareLinkLoss,
	prCommitUrlRegex,
	prCompareUrlRegex
```